apps/channel/client/channel_view.coffee

```diff
@@ -41,14 +41,14 @@ module.exports = class ChannelView extends Backbone.View
     collaborator = _.contains collaborators.pluck('id'), mediator.shared.current_user.id
 
     # Addable
-    if collaborator or mediator.shared.current_user.canAddToChannel(@channel) and not (FOLLOWERS? or FOLLOWING?)
+    if CAN.add_to
       @setupFileDropView()
 
       @$('.js-block-collection').addClass 'is-addable'
       mediator.trigger 'channel:is-addable'
 
     # Editable
-    if collaborator or mediator.shared.current_user.canEditChannel(@channel)
+    if CAN.update
       @$('.js-block-collection').addClass 'is-editable'
       @$('.block-item').addClass 'can-manage'
 
```

Comment:
I think I'll need to add some new abilities for the stuff below here. Still trying to make some sense of it.